### PR TITLE
Changing the clientID for FreeCodeCamp

### PR DESCRIPTION
**Pull request by tom-logging**

### CODE REVIEW

File: websites/F/FreeCodeCamp/dist/metadata.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "author": {
     "name": "tom",
-    "id": "266024212978728963"
+    "id": "790880432954409000"
```
Comment (ririxidev) [inline websites/F/FreeCodeCamp/dist/metadata.json:4]: this is id of your account, not clientid of presence...

**Review by ririxidev — CHANGES_REQUESTED** [review]
this and bump version and revert changes and change clientid in presence.ts? and upload proof of working presence

**Comment by tom-logging** [comment]
> this and bump version and revert changes and change clientid in presence.ts? and upload proof of working presence

Today isn't my day, I meant to change it in the presence not metadata

**Comment by Bas950** [comment]
> > this and bump version and revert changes and change clientid in presence.ts? and upload proof of working presence
> 
> Today isn't my day, I meant to change it in the presence not metadata

Just make a new PR once you have fixed it.

**Pull request closed without merging**